crt_portal/cts_forms/forms.py

```diff
@@ -145,7 +145,8 @@ def __init__(self, *args, **kwargs):
             error_messages={
                 'required': PRIMARY_COMPLAINT_ERROR
             },
-            help_text=_('Please choose the option below that best fits your situation. The examples listed in each are only a sampling of related issues. You will have space to explain in detail later.')
+            label=_('What is the primary reason for contacting the Civil Rights Division'),
```

Comment:
From the ticket
image says: 
"What is **the** primary reason **you're** contacting the Civil Rights Division?" 
the card says: 
"What is **your** primary reason **for** contacting the Civil Rights Division?"

The [content source of truth](https://docs.google.com/document/d/1bxSRc9CQOHjQhTaYkHncE4tykUEFeC6RzFtP5IQk0Ic/edit#heading=h.9dghwdsa1dic) has it the same as the card- so let's go with that and update later as needed.

```suggestion
            label=_('What is your primary reason for contacting the Civil Rights Division?'),
```

Reply:
all updated! I will eventually consistently check the content source of truth, eventually 😬

Reply:
No worries!